### DISCUSSION TITLE
pre_capture specific for objects?

### DISCUSSION THREAD

**Discussion by Yonny24** [blakeblackshear/frigate#7204] [Ask A Question] [answered]
Is there a way of defining the pre-capture at the objects level by any chance?

For example cat pre-capture 5 but person only 2?

Default in mine:

  events:
    # Optional: Maximum length of time to retain video during long events. (default: shown below)
    # NOTE: If an object is being tracked for longer than this amount of time, the retained recordings
    #       will be the last x seconds of the event unless retain_days under record is > 0.
    #max_seconds: 10
    # Optional: Number of seconds before the event to include (default: shown below)
    pre_capture: 1
    # Optional: Number of seconds after the event to include (default: shown below)
    post_capture: 4
    # Optional: Objects to save recordings for. (default: all tracked objects)
    objects:
      - person
      - cat 
      - dog
      - motorcycle
      
      Thanks!

**Comment by NickM-27** [comment]
It's not currently possible.

**Reply by Yonny24** [reply, marked as answer]
ah ok - thanks for your rapid replies.

Cheers
